README.md

```diff
@@ -156,10 +150,7 @@ kubectl create -f wavefront-advanced-proxy.yaml
 
 ##### Note on Configuration Precedence
 
-If you include a redundant configuration of a proxy arg
-in both the dedicated Custom Resource field
-and via the command line argument input in `dataExport.wavefrontProxy.args`,
-the dedicated Custom Resource field will take precedence.
+All wavefront proxy configuration options should be available via explict fields in the custom resource. That is our recommendation. However, it is also possible to specify options as proxy command line parameters in the `dataExport.wavefrontProxy.args` field. If both are used, the operator will allow it at this time, but the custom resource field will take precidence.
```

Comment:
Some wavefront proxy configurations options are supported today (not all). We only support those proxy configurations which are currently supported via collector/proxy helm.

Reply:
I believe we support all proxy configurations since the user can add any additional proxy args via dataExport.wavefrontProxy.args

Reply:
Cool, I figured someone would fact check this.